h/models/feature_cohort.py

```diff
@@ -22,16 +22,14 @@ def __init__(self, name):
         self.name = name
 
 
-FEATURECOHORT_USER_TABLE = sa.Table(
-    "featurecohort_user",
-    Base.metadata,
-    sa.Column("id", sa.Integer, nullable=False, autoincrement=True, primary_key=True),
-    sa.Column(
-        "cohort_id", sa.Integer, sa.ForeignKey("featurecohort.id"), nullable=False
-    ),
-    sa.Column("user_id", sa.Integer, sa.ForeignKey("user.id"), nullable=False),
-    sa.UniqueConstraint("cohort_id", "user_id"),
-)
+class FeatureCohortUser(Base):
+    __tablename__ = "featurecohort_user"
+    __table_args__ = (sa.UniqueConstraint("cohort_id", "user_id"),)
+
+    id = sa.Column(sa.Integer, nullable=False, autoincrement=True, primary_key=True)
+    cohort_id = sa.Column(sa.Integer, sa.ForeignKey("featurecohort.id"), nullable=False)
+    user_id = sa.Column(sa.Integer, sa.ForeignKey("user.id"), nullable=False)
```

Comment:
I need this to be an ORM class so that I can delete rows from this table using the same code as I use to delete rows from other tables via their ORM classes.